contracts/fixed_price_rev1.1.scilla

```diff
@@ -131,7 +144,13 @@ let make_error =
 
 contract FixedPriceOrderbook
 (
-  initial_contract_owner: ByStr20
+  initial_contract_owner: ByStr20,
+  collection_contract: ByStr20 with contract
```

Comment:
@sam-zilliqa can we use `initial_collection_contract` instead?

Reply:
Yes, updated